src/serializer.cpp

```diff
@@ -88,6 +88,21 @@ std::string realastic_impl::serialize_open_event(struct_open_syscall *event)
     return ss.str();
 }
 
+std::string realastic_impl::serialize_creat_event(struct_creat_syscall *event)
+{
+    std::string buff;
+    if (event->rc > 0)
+    {
+        buff = std::move(escape_special_charachter(std::string(event->pathname)));
```

Comment:
seems I made a mistake in function name spells, please refactor charchter to character. Tnx